RELEASE.md

```diff
@@ -2,18 +2,78 @@
 
 This document guides a contributor through creating a release of JupyterLab.
 
+## Automated Releases with the Jupyter Releaser
+
+The recommended way to make a release is to use [`jupyter_releaser`](https://github.com/jupyter-server/jupyter_releaser#checklist-for-adoption).
+
+### Workflow
+
+The full process is documented in https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_first_release.html#making-your-first-release.
+
+Here is a quick summary of the different steps.
+
+#### Draft Changelog
+
+The first step is to generate a new changelog entry for the upcoming release.
+
+We use the "Draft Changelog" workflow as documented here: https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_first_release.html#draft-changelog
+
+The workflow takes a couple of input parameters, for example:
+
+| Input        | Value                 |
+| ------------ | --------------------- |
+| Target       | jupyterlab/jupyterlab |
+| Branch       | master                |
+| Version Spec | next                  |
+| Since        | v4.0.0a15             |
+
+The version spec follows the specification documented bellow in the [Bump Version](#bump-version) section.
+
+We can use `next` when making a `patch` release or a `build` pre-release.
+
+Here is an example screenshot of what it should look like:
+
+![draft-changelog-example](https://user-images.githubusercontent.com/591645/141277239-571c2e6a-e8d5-4af9-aee4-dd52a7dae8ba.png)
+
+We then wait for:
+
+1. the PR to be created on the repo. Example: https://github.com/jupyterlab/jupyterlab/pull/11422
+2. Tests to pass
+3. Merge the changelog PR
+
+### Full Release
```

Comment:
Can you add a note about making sure to set up up NPM and PYPI tokens before running the release workflows?

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/11440/commits/bfae1222f03369f9eb5c87ee0479daec3c388db0